docs/content/latest/api/ysql/syntax_resources/the-sql-language/statements/delete,returning_clause.grammar.md

```diff
@@ -1,6 +1,6 @@
 ```
 delete ::= [ WITH [ RECURSIVE ] { common_table_expression [ , ... ] } ] 
-            DELETE FROM [ ONLY ] table_name [ * ] [ [ AS ] alias ]  
+            DELETE FROM table_expr [ [ AS ] alias ]  
```

Comment:
Is there a way to link these definitions? (Since `common_table_expression` and `table_expr` do have their own definitions...)

Reply:
Read this note, immediately following the diagram:

> The _[table_expr](http://localhost:1313/latest/api/ysql/syntax_resources/grammar_diagrams/#table-expr)_ rule specifies syntax that is useful only when at least one other table inherits one of the tables that the truncate statement lists explicitly. See [this note](http://localhost:1313/latest/api/ysql/the-sql-language/statements/ddl_alter_table#table-expr-note) for more detail. Until inheritance is supported, use a bare _[table_name](http://localhost:1313/latest/api/ysql/syntax_resources/grammar_diagrams/#table-name)_.

You can click on _table_expr_ to see the diagram in the reference compendium of all diagrams [HERE](https://docs.yugabyte.com/latest/api/ysql/syntax_resources/grammar_diagrams/#abort). I did have the choice to include the _table_expr_ diagram right there on the _DELETE_ page with the other two that are presently included. But I decided not do do that because I recommend against using that syntax that it allows. we can revisit this when YSQL eventually support table inheritance.